pom.xml

```diff
@@ -10,16 +10,16 @@
 	
 	<groupId>eu.arrowhead</groupId>	
 	<artifactId>client-library</artifactId>
-	<version>4.1.3.13</version>
+	<version>4.4.1</version>
```

Comment:
we should use 4.4.0.1 where 4.4.0 is referring to the core-java-spring (which version the core-client-skeleton came from)